php/Terminus/Models/User.php

```diff
@@ -80,9 +79,9 @@ public function getOrganizations() {
    */
   public function getSites($organization = null) {
```

Comment:
Is this function still being used? I'm not sure why it's not accessing a `UserSiteMemberships` collection (unless that just hasn't been written).